strategies/BidAskSpreadMA/consolidating-MA-buyside-wmaticuusdt.rain

```diff
@@ -178,12 +184,11 @@ current-time: block-timestamp(),
 
 /*set conditions*/
 buycondition: every(
-  is-zero(get("trimeme-26-has-buy-binary")) 
+  is-zero(get("trimeme-24-has-buy-binary")) 
   greater-than(current-price range-floor) 
   less-than(current-price range-ceiling)),
 
 /*price params*/
```

Comment:
expect `price-needed`, the rest of the elemets of the price params are irrelevant to the execution of the order, so they should be on the plottables stack within the plot binding and not in prod.